tests/lwe_to_polynomial/ops_dimension_error.mlir

```diff
@@ -8,8 +8,9 @@
 !invalid_ciphertext = !lwe.rlwe_ciphertext<encoding = #encoding, rlwe_params = #invalid_rlwe_params, underlying_type=i3>
 !invalid_secret_key = !lwe.rlwe_secret_key<rlwe_params=#invalid_rlwe_params>
 
-// expected-error@below {{Expected 2 dimensional ciphertext, found ciphertext tensor dimension = 10}}
 func.func @invalid_dimensions_rlwe_decrypt(%arg0: !invalid_ciphertext, %arg1: !invalid_secret_key) -> !plaintext_rlwe {
+// expected-error@below {{Expected 2 dimensional ciphertext, found ciphertext tensor dimension = 10}}
+// expected-error@below {{failed to legalize operation 'lwe.rlwe_decrypt'}}
```

Comment:
nit: adjust spacing before `// expected-error` to match spacing of the rest of the function.